dashboard/src/components/DeploymentForm/DeploymentFormBody/BasicDeploymentForm/TabularSchemaEditorTable/TabularSchemaEditorTableHelpers.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+import { compareItems, rankItem } from "@tanstack/match-sorter-utils";
+import { FilterFn, SortingFn, sortingFns } from "@tanstack/react-table";
```

Comment:
In this file, we are just exposing the filtering functions as per the docs: https://tanstack.com/table/v8/docs/examples/react/filters